### PR TITLE
[canvaskit] default to Ahem in flutter_tester emulation mode

### PR DESCRIPTION
**Pull request by yjbanov**
Ignore the `fontFamily` and `fontFamilyFallbacks` passed by the framework and always use Ahem when emulating the `flutter_tester` environment.

### CODE REVIEW

File: lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      );
+
+      await matchPictureGolden(
+        'canvaskit_defaults_to_ahem.png',
+        recorder.endRecording(),
+        region: kDefaultRegion,
+      );
+    });
+    // TODO(yjbanov): https://github.com/flutter/flutter/issues/60040
+    // TODO(yjbanov): https://github.com/flutter/flutter/issues/71520
```
Comment (mdebbar) [inline lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart:116]: Do we even run canvaskit tests on other browsers?

**Review by mdebbar — APPROVED** [review]
LGTM!

File: lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart (hunk 1 above)
Comment (yjbanov) [reply lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart:116]: Yes, we run in Chrome, Safari desktop, and Firefox, but we screenshot only on Chrome.

Comment (mdebbar) [reply lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart:116]: Ah okay that makes sense cuz I only saw Canvaskit/Chrome screenshots in Gold.

Comment (yjbanov) [reply lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart:116]: We can take screenshots from Safari for iOS already. The only thing that's stopping us is that LUCI needs to upgrade the iOS Simulator to 14 or newer. We're still testing on iOS 13.

Comment (yjbanov) [reply lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart:116]: (iOS Simultaor 14 added support for WebAssembly, which is what's actually blocking)

**Pull request merged**